block_function.rb

```diff
@@ -0,0 +1,14 @@
+def do_calc
+  yield 7, 9
+  yield 7, 9
```

Comment:
`Yield` keyword will run all code blocks in this file if no specific block/method assigned to it.
We only need one line of yield to pass 7 & 9 into the following methods (since we yield to TWO methods in the same file), so it won't print the same output twice when you call the file on the command line.